makefile.shade

```diff
@@ -30,6 +30,7 @@ k-standard-goals
       "-n ResponseCachingPlaintextCached",
       "-n ResponseCachingPlaintextResponseNoCache",
       "-n ResponseCachingPlaintextRequestNoCache",
+      "-n ResponseCachingPlaintextVaryByCached",
```

Comment:
One more test is fine if we think it will be useful.  We can always remove tests if the run starts taking too long.